ucm2/sof-soundwire/rt715.conf

```diff
@@ -5,10 +5,12 @@ SectionDevice."Mic" {
 
 	EnableSequence [
 		cset "name='PGA5.0 5 Master Capture Switch' 1"
+        cset "name='rt715 Micmute Led Mode' On"
```

Comment:
it looks like this is using spaces, but the file normally uses tabs to me.

Reply:
I will update the patch, check later.